src/lib/MovingAverageInterface.java

```diff
@@ -9,7 +9,8 @@
 public interface MovingAverageInterface {
 
   /**
-   * Removes the first value from the average ands a new one to the average.
+   * Removes the first value from the average and adds a new one to the average. A value is only
```

Comment:
This could be sharper and more direct:

"Adds the new value to the average and removes the least recent value if there would be more than windowSize values"